packages/stark-ui/src/modules/restrict-input-directive.ts

```diff
@@ -0,0 +1 @@
+export * from "./restrict-input-directive/restrict-input-directives.module";
```

Comment:
Can you put the directive under a `directives` sub-folder as we do in the other modules so we keep consistency?